src/connectors/hits-per-page/connectHitsPerPage.js

```diff
@@ -167,10 +170,19 @@ Learn more: https://community.algolia.com/instantsearch.js/v2/widgets/configure.
             ? helper.setQueryParameter('hitsPerPage', undefined).search()
             : helper.setQueryParameter('hitsPerPage', value).search();
 
+        createURLFn = value =>
```

Comment:
For the moment IMO it's easier to follow the code with the `this` binding. Until we have a proper API to support it I think we should stick to the pattern than we used almost everywhere.

Reply:
Yes – I originally went for this because of #3347.